src/pages/MyInfo/MyInfo.js

```diff
@@ -1,12 +1,152 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import InvestSummary from './InvestSummary';
+import Deposit from './Deposit';
 
 function MyInfo() {
+  const [tabIndex, setTabIndex] = useState(1);
+  const [myInfo, setMyInfo] = useState({});
+
+  useEffect(() => {
+    fetch('/data/myInfo.json')
+      .then(res => res.json())
+      .then(data => {
+        setMyInfo(data.results);
+      });
+  }, []);
+
+  const MAPPING_MyInfoTab = {
+    1: (
+      <InvestSummary
+        deposit={myInfo.deposit}
+        investing={myInfo.investing}
+        investmentCompleted={myInfo.investmentCompleted}
+        loss={myInfo.loss}
+        investingTab={myInfo.investingTab}
+        accountNumber={myInfo.accountNumber}
+        goodsTab={myInfo.goodsTab}
+        yieldTab={myInfo.yieldTab}
+      />
```

Comment:
```suggestion
      <InvestSummary
        myInfo={myInfo}
      />
```
myInfo 전체 데이터를 보내주고 자식컴포넌트에서는 비구조화할당을 사용하시면 더 효율적이겠네요!